deploy/apache/sites-available/privacyidea.conf

```diff
@@ -55,3 +63,15 @@
 
 
 </VirtualHost>
+
+<VirtualHost _default_:80>
```

Comment:
Cornelius Kölbel <notifications@github.com> writes:

>> @@ -55,3 +63,15 @@
>>  
>>  
>>  </VirtualHost>
>> +
>> +<VirtualHost _default_:80>
>
> I would rather not have this port 80 in this file by default.
> Can you please remove this?

Should we remove port 80 from nginx too? We have this:

server {
        listen  80;
        server_name     $hostname;
        return 301 https://$hostname$request_uri;
}

I think we should try to keep nginx and apache configuration
similar. And I personally find it useful that I get redirected to the
working https-site instead of an empty index page when calling
http://pi.example.org.

I see three possible ways forward:
1. both nginx and apache forward me from http to https
2. both servers don't have port 80 at all
3. ngingx does, but apache doesn't (that's what we have now)

Just confirm you really want 3 and I'll amend the commit.

-- 
The only problem with troubleshooting is that the trouble shoots back.